requirements.txt

```diff
@@ -102,7 +102,7 @@ python-redis-lock==3.7.0
 python-swiftclient==4.4.0
 pytz==2019.3
 PyYAML==6.0.2
-raven==6.10.0
+sentry-sdk[flask]==1.40.0
```

Comment:
curious, do we need `[flask]` bit? not sure if it's compatible with downstream builds